src/App.tsx

```diff
@@ -65,8 +63,21 @@ function App() {
     );
   }
 
+  const apolloClient = new ApolloClient({
+    uri: `${apiUrl}/query`,
+    request: (operation) => {
+      const token = user.AccessToken;
```

Comment:
what is `user` in this context? where is it defined?

Reply:
It is a state variable declared on line 29. It is set by a call to `setUser` in `signIn` on line 39.